### ISSUE TITLE
Object.extname is deprecated.

### ISSUE THREAD

**Issue by dc-avasilev** [mulesoft/api-workbench#410]
Argument to `path.extname` must be a string
```
Object.extname (/usr/share/atom/resources/app.asar/src/electron-shims.js:20:10)
e.calculate (/home/$USER/.atom/packages/api-workbench/main.js:3:9362)
<unknown> (/home/$USER/.atom/packages/api-workbench/node_modules/raml-actions/dist/actionManagement/contextActionsImpl.js:74:52)
Array.forEach (native)
Object.calculateCurrentActions (/home/$USER/.atom/packages/api-workbench/node_modules/raml-actions/dist/actionManagement/contextActionsImpl.js:70:25)
Object.calculateCurrentActions (/home/$USER/.atom/packages/api-workbench/node_modules/raml-actions/dist/index.js:21:31)
```

**Comment by dc-avasilev** [comment]
atom is old version, help update... sorry

**Comment by dc-avasilev** [comment]
hm... noup, not help, im wrong

**Comment by dc-avasilev** [comment]
problem is actual

**Comment by ddenisenko** [comment]
Could you please provide reproduction steps?

**Comment by dc-avasilev** [comment]
i just open work yaml project

**Comment by dc-avasilev** [comment]
![screenshot at 2017-02-14 19-41-18](https://cloud.githubusercontent.com/assets/17141669/22938695/98ed6244-f2ed-11e6-8de8-24cb85c7e310.png)

**Comment by ddenisenko** [comment]
Thank you.

**Comment by dreamflyer** [comment]
will be published in v0.8.44.
